source/sink-connector/configuration-properties.txt

```diff
@@ -22,6 +22,10 @@ settings.
 For an example sink connector configuration file, see
 :github:`MongoSinkConnector.properties <mongodb/mongo-kafka/blob/{+connector_version_github_tag+}/config/MongoSinkConnector.properties>`.
 
+For some performance tips and general recommendations for configuring your
```

Comment:
Suggestion:

Technically, the linked article doesn't contain "performance tips... for configuring" but rather the "configurations for performance". I think this can be more succinct by avoiding qualifiers like "some" and "general" (may not be in the style guide, but don't think my suggestion below changes the meaning):

```suggestion
For sink connector performance recommendations, see ...
```